src/ucp/wireup/select.c

```diff
@@ -271,6 +265,10 @@ ucp_wireup_select_transport(ucp_ep_h ep, const ucp_address_entry_t *address_list
     p                   = tls_info;
     endp                = tls_info + sizeof(tls_info) - 1;
     tls_info[0]         = '\0';
+    tl_bitmap          &= select_ctx->tl_bitmap;
+    show_error          = (select_ctx->show_error && show_error);
```

Comment:
minor: here you can also do `&=` as on the line above

Reply:
pls note, we do `&&`. `show_error` should be `1`, if both `select_ctx->show_error` and `show_error` are `1`
it would work, but (imo) it will make code unclear

Reply:
yes, you are right, missed the second `&`